docs/rst/tutorials/core/qos/initial_peers/initial_peers.rst

```diff
@@ -0,0 +1,219 @@
+| <container ID1> | ubuntu-vulcanexus:humble-desktop | "…vulcanexus…" | 2 minutes ago | Up 2 minutes |       | keen_john  |
++-----------------+----------------------------------+----------------+---------------+--------------+-------+------------+
+| <container ID2> | ubuntu-vulcanexus:humble-desktop | "…vulcanexus…" | 2 minutes ago | Up 2 minutes |       | eager_pike |
++-----------------+----------------------------------+----------------+---------------+--------------+-------+------------+
+
+.. important::
+
+    Check the container ID of the docker instance where the XML was created, and obtain the **other** docker instance IP address.
+
+
+Using the provided auto-generated instance name (last column content in the output table), check its detailed information.
+The following command also filters the output to obtain only the instance IP address:
+
+.. code-block:: bash
+
+      docker inspect <instance_autogenerated_name(keen_jones)> | grep IPAddress
+
+
+Edit the previously created XML file, and include the IP address of the **other** instance.
+
+.. important::
+
+    Make sure the IP address set in the xml document is NOT the instance's own IP address.
+
+.. _initial_peers_xml_location:
+
+XML configuration file location
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to use the profiles, it is required to let the application know it.
+There are two possibilities for providing *Fast DDS* with XML configuration files:
+
+* **Recommended**: Setting the location with environment variable ``FASTRTPS_DEFAULT_PROFILES_FILE`` to contain the path to the XML configuration file (see `Environment Variables <https://fast-dds.docs.eprosima.com/en/latest/fastdds/env_vars/env_vars.html#env-vars>`_).
+
+.. code-block:: bash
+
+      export FASTRTPS_DEFAULT_PROFILES_FILE=<path_to_xml_file>
+
+* **Alternative**: Placing the XML file in the running application directory under the name *DEFAULT_FASTRTPS_PROFILES.xml*.
+
+For example:
+
+.. code-block:: bash
+
+    cp <path_to_xml_file> DEFAULT_FASTRTPS_PROFILES.xml
+    export RMW_FASTRTPS_USE_QOS_FROM_XML=1
+
+.. important::
+
+    Repeat the same procedure in the other opened Vulcanexus docker instance.
+
+.. _initial_peers_example:
+
+Example
+-------
+
+Once both Vulcanexus docker environments have been configured, run a demo application in each instance:
+
+.. code-block:: bash
+
+      # Talker
+      ros2 run demo_nodes_cpp talker
+
+      # Listener
+      ros2 run demo_nodes_cpp listener
+
+The Vulcanexus docker instances would discover each other via multicast, then one of them would let the other know its address and port through it would establish any communication, and they will connect.
+
+.. _initial_peers_unicast_example:
+
+Unicast Example
+^^^^^^^^^^^^^^^
+
+If unicast is required instead of multicast, the following profile should be updated in the existing XML profile:
```

Comment:
```suggestion
In the previous example, both multicast and unicast communication were used for entity discovery. Next, it is shown how to fully disable multicast communication for discovery, so that the only means of discovery is through unicast communication. In this case, one of the participants will know in advance the listening address of the other participant and will send its discovery information via unicast communication. The second participant will then reply to the first participant using the network address contained in the information received. After the discovery, both will start the transmission of user data. 

Replace the content of the existing XML configuration file with the one shown below.
```